pptx_blueprint/__init__.py

```diff
@@ -37,7 +37,27 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
             label (str): label of the placeholder (without curly braces)
             filename (path-like): path to an image file
         """
-        pass
+        shapes_to_replace = self._find_shapes(label)
+        if not shapes_to_replace:
+            return
+        if isinstance(filename, str):
+            filename = pathlib.Path(filename)
+        if not filename.is_file():
+            raise FileNotFoundError(f"The file does not exist: {filename}")
+        img_file = open(filename, "rb")
+        old_shape: BaseShape
+        for old_shape in shapes_to_replace:
+            slide_shapes = old_shape._parent
```

Comment:
Not a big fan of using the internals of the pptx library. But if there is really not better way...

Reply:
It seems there is currently no other solution:
https://github.com/scanny/python-pptx/issues/246

Could be mentionend in a code comment.

Reply:
Thx for linking the resource.